crates/sui-cost/tests/snapshots/empirical_transaction_cost__good_snapshot-2.snap

```diff
@@ -4,13 +4,13 @@ expression: common_costs_estimate
 ---
 {
   "MergeCoin": {
-    "computation_cost": 4983,
-    "storage_cost": 5870,
+    "computation_cost": 5218,
```

Comment:
Are these changes expected? Doesn't seem like this PR should be affecting these functions

Reply:
I'm not sure but this is usually what happens with every update of the std.

Reply:
There's a small fix in `sui::crypto` as well but it should not affect these.

Reply:
Ah, it's because the framework package is now bigger